app/src/app/pages/artist/artist.component.html

```diff
@@ -3,42 +3,20 @@
     <div class="col-sm-7" *ngIf="artist">
       <app-title [entity]="artist"></app-title>
 
-      <table class="table-fit">
-        <tbody>
-          <tr *ngIf="artist?.dateOfBirth">
-            <td class="text-muted"><ng-container i18n="Born infos @@born">Born</ng-container>:</td>
-            <td>{{ artist?.dateOfBirth }}</td>
-          </tr>
+      <app-information label="Geboren" [value]="artist?.birth"></app-information>
+      <app-information label="Gestorben" [value]="artist?.death"></app-information>
```

Comment:
Please keep this in english, the german translation needs to stay in the translation file.